tests/smoke/aws/README.md

```diff
@@ -20,7 +20,7 @@ To begin, verify that the following environment variables are set:
 
 - `AWS_PROFILE` or alternatively `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`: These credentials are used by Terraform to spawn clusters on AWS.
 - `TF_VAR_tectonic_pull_secret_path` and `TF_VAR_tectonic_license_path`: The local path to the pull secret and Tectonic license file.
-- `TF_VAR_tectonic_aws_ssh_key`: The AWS ssh key pair which enables ssh'ing into the created machines using the `core` user.
+- `TF_VAR_tectonic_admin_ssh_key`: The SSH public key which enables SSHing into the created machines using the `core` user.
```

Comment:
The new setting should be optional, so this line can grow an `(optional)` prefix (like the later list entries).